vespajlib/src/main/java/com/yahoo/concurrent/classlock/ClassLocking.java

```diff
@@ -34,4 +44,10 @@ void unlock(Class<?> clazz, ClassLock classLock) {
             }
         }
     }
+
```

Comment:
I have no better place to put this comment than here. The unlock() method tests for equality of classLock with classLocks.get(), and otherwise throws "Lock has already been released".

It could throw one exception for double-unlock, and another for not owning the lock.

Reply:
The idea was that if you own the lock instance, you own the lock. The instance is only created when you acquire the lock, therefore, if you try to unlock with an instance not in the map, then the lock must already have been released, since it's not possible to create your own lock (package-private constructor).

Exception: `null`.

Reply:
ok